app/components/Layout/Layout.js

```diff
@@ -84,7 +85,12 @@ class Layout extends React.Component {
         {this.props.notifications}
 
         <Flex auto>
-          {showSidebar && <Sidebar />}
+          {showSidebar && (
+            <Switch>
+              <Route path="/settings" component={SettingsSidebar} />
```

Comment:
All settings routes have to be under `/settings` for this to work, seems reasonable though.